src/lando/main/models/worker.py

```diff
@@ -27,7 +27,16 @@ class Worker(BaseModel):
     )
 
     def __str__(self):
-        return self.name
+        if self.is_paused:
+            state = "PAUSED"
+        elif self.is_stopped:
+            state = "STOPPED"
+        else:
+            state = "RUNNING"
```

Comment:
Nit: Would it make sense to have a single `state` value, rather than separate booleans? All three seem mutually exclusive.